### ISSUE TITLE
Error initializing output widget in notebook when comm does not exist

### ISSUE THREAD

**Issue by pbugnion** [jupyter-widgets/ipywidgets#1598]
To trigger the bug, 

1. create an output widget with, e.g.:

```python
from ipywidgets import interact
def f(x):
    return x
interact(f, x=10)
```

2. save the widget state (using the *Widgets* > *Save Notebook Widget State* menu entry).

3. restart the kernel

4. reload the browser client by reloading the page

This triggers an error `Uncaught (in promise) TypeError: Cannot read property 'kernel' of undefined`. AFAICT, the error is caused [here](https://github.com/jupyter-widgets/ipywidgets/blob/master/widgetsnbextension/src/widget_output.js#L24). We dereference `this.comm`, but `this.comm` can be null. 

Happy to look into this this week, unless someone gets there first.

**Comment by jasongrout** [comment]
> Happy to look into this this week, unless someone gets there first.

+1

**Comment by jasongrout** [comment]
Fixed by #1605.
